src/main/java/org/mule/modules/git/GitConnector.java

```diff
@@ -0,0 +1,419 @@
+package org.mule.modules.git;
```

Comment:
Please keep org.mule.module.git in order to preserve file history. Then change the package to org.mule.modules.git in another commit.